compiler/lsp-tests/src/Main.hs

```diff
@@ -816,7 +816,6 @@ completionTests run _runScenarios = testGroup "completion"
               map (set documentation Nothing) completions @?=
               [ mkTypeCompletion "Party"
               , mkTypeCompletion "IsParties"
-              , mkTypeCompletion "TemplateOrInterface"
```

Comment:
I am not sure, why I have do not have to replace `TemplateOrInterface` by `Template` here.

Reply:
I think this completion simply triggered because of `TemplateOrInterface` has all the letters in `Part` in order,

```
TemplateOrInterface
   p a   r  t
```

though I wasn't able to get this completion in Daml Studio. In any case, it's okay to remove it.

Reply:
I got it.